### PR TITLE
Note support status of Project and Smart Proxies

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1823868 - Need to clarify that Multi-homed satellite setup is not supported

https://bugzilla.redhat.com/show_bug.cgi?id=1823868

### CODE REVIEW

File: guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc [hunk 1]
```diff
@@ -28,10 +28,10 @@ You can create additional organizations and then import additional manifests to
 endif::[]
 
 .Multi-homed {Project} and {SmartProxy}
-If you want {Project} to use multiple interfaces with distinct host names, you must perform additional configuration of the {ProjectServer} and {Project} {SmartProxyServer} CA certificates.
-ifeval::["{build}" == "satellite"]
-If you want to deploy {Project} in this configuration, contact Red Hat.
-endif::[]
+
+Multi-homed {Project} is not supported.
```
Comment (ekohl) [inline guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc:32]: I would keep this as a Satellite specific note. Support status is really a downstream only thing.

File: guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc [hunk 2]
```diff
@@ -28,10 +28,10 @@ You can create additional organizations and then import additional manifests to
...
+
+Multi-homed {SmartProxy} is supported, to implement this, you can configure {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```
Comment (ekohl) [inline guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc:34]: This too. In upstream you can run a multi-homed Smart Proxy (also without a load balancer) but downstream doesn't want to support it.

**Review by ekohl — COMMENTED** [review]
I didn't really read this document before, but the sentence:
> In AWS, you cannot manage the DHCP. Because of this, most of {ProjectServer}’s kickstart and PXE provisioning models are unusable. This includes:

You probably can't provision using PXE inside AWS itself, but a Foreman server in AWS with a Smart Proxy in your own Data Center should work. That's IMHO the whole reason you have Smart Proxies.

File: guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc [hunk 3]
```diff
@@ -29,9 +29,15 @@ endif::[]
...
 .Multi-homed {Project} and {SmartProxy}
 
+ifeval::["{build}" == "satellite"]
 Multi-homed {Project} is not supported.
 
 Multi-homed {SmartProxy} is supported, to implement this, you can configure {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+You can use multi-homed {SmartProxies} by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```
Comment (ekohl) [inline guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc:39]: This implies it's the only way. Would something like this be more appropriate?
```suggestion
One way to use multi-homed {SmartProxies} is by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```

Comment (spetrosi) [reply guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc:39]: Sure this works perfect!

**Review by ekohl — APPROVED** [review]

**Comment by suyogsainkar** [comment]
@spetrosi looks good to me; merging.

**Pull request merged**